### DISCUSSION TITLE
[Feature] Mobile switching organization

### DISCUSSION THREAD

**Discussion by a20070322** [immich-app/immich#11368] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

At present, my immich is set up on the local area network to avoid the data cost generated by backup, so I synchronize the images in iOS on the local area network. Not browsing pictures at home requires switching to a public address. But currently I cannot store the accounts I have logged in to, and it is very troublesome to re-enter them every time. I look forward to you implementing this feature

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2919
